.github/workflows/code-linting-checks.yaml

```
@@ -0,0 +1,40 @@
+name: "code-linting-checks"
+
+on:
+  pull_request:
+    paths: ["**/*.yaml", "**/*.yml"]
+  push:
+    paths: ["**/*.yaml", "**/*.yml"]
```

```suggestion
    paths: ["**/*.yaml"]
  push:
    paths: ["**/*.yaml"]
```

Since the `.yml` is disallowed by our [guidelines](https://www.notion.so/yscope/WIP-Coding-Guidelines-9a308b847a5343958ba3cb97a850be66?pvs=4#28ee6f2e3a2243b48be4084df825fce1), let's not include it.